### DISCUSSION TITLE
Is it possible to hide or disable TODO timestamp when complited?

### DISCUSSION THREAD

**Discussion by KorbenOP** [orgzly-revived/orgzly-android-revived#181] [Q&A] [answered]
Is there a way to hide them in UI, or completely disable them, after a task has been completed?

**Comment by amberin** [comment]
Not sure exactly what you mean. But you can always remove the TODO keyword by tapping the flag symbol and clearing its value.

**Reply by KorbenOP** [reply]
I meant that when a note goes to the "completed" state, a property appears with the date and time when it was completed. And this property is displayed in the list of notes, taking up space and creating unnecessary information. And wondering if there is a way to disable this or hide it. I found an item in the settings that hides all dates: scheduled, deadlines, etc, but that's not what I need. The only way is to go into the note every time and manually delete this property.
![2024-02-24_091958](https://github.com/orgzly-revived/orgzly-android-revived/assets/25414496/12a961e8-02ac-478e-9ded-eddf698eac74)
![2024-02-24_092043](https://github.com/orgzly-revived/orgzly-android-revived/assets/25414496/46928ea8-3869-461f-a115-5fc51c418a17)

**Comment by amberin** [comment, marked as answer]
I see, thank you for explaining. I suppose it shouldbe easy to add a setting for hiding that property from the note list. Either that or disabling adding the property, but my guess is that most users want the information, even if it's not shown in the note list (after all, that is the org-mode experience).

**Reply by KorbenOP** [reply]
I understand, thanks for the answer. I switched from Dynalist, so this default behavior was a little unusual for me.

**Reply by alensiljak** [reply]
Yes, Orgzly (as the name implies) draws concepts from OrgMode. I would suggest to check it out. It is not just a todo list that removes done items, but rather a project management app/system.
